src/Units/Weapons/Damage/StandardDamage.java

```diff
@@ -0,0 +1,43 @@
+package Units.Weapons.Damage;
+
+import Units.UnitModel;
+import Units.Weapons.WeaponModel;
+
+public class StandardDamage extends DamageStrategy
```

Comment:
Standard damage is what, AW2? Might be better to just call it that, since the others are named more specifically.